test/lib/plugins/send-email-with-mandrill.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const sendEmailWithMandrill = require('../../../lib/plugins').sendEmailWithMandrill;
+
+describe('sendEmailWithMandrill', () => {
+  describe('composeEmail', () => {
```

Comment:
Would be good to test the Markdown to HTML renderer too, if possible. Just to make sure it doesn't die due to invalid markdown in template (which is likely to happen when modifying the template).